platform-friends-hybrid/styles/main.css

```diff
@@ -32,585 +46,610 @@
 .km-flat li.km-state-active .km-listview-label,
 .km-flat .km-listview-label input[type=radio]:checked,
 .km-flat .km-listview-label input[type=checkbox]:checked {
-    background: #3598db;
+  background: #4d4d4d;
+}
+
+.km-flat .km-loader {
+  background-color: rgba(0, 0, 0, .75);
+}
+
+.km-flat .km-loader::before {
+  background: none;
+}
+
+#friendsApp.km-flat,
+#friendsApp.km-flat .km-navbar,
+#friendsApp.km-flat .km-content,
+input[type='date'],
+input[type='datetime-local'],
+input[type='month'],
+input[type='time'],
+input[type='week'] {
+  font-family: 'Roboto', sans-serif;
 }
 
 #friendsApp.km-flat,
 #friendsApp.km-flat .km-navbar,
 #friendsApp.km-flat .km-content {
-    border: 0 none;
-    background-color: #2d3e50;
-    color: #fff;
-    font-family: "Open Sans", sans-serif;
+  border: 0 none;
+  color: #4d4d4d;
+  background-color: #fff;
 }
 
-#friendsApp .center-block {
-    display: block;
-    margin-right: auto;
-    margin-left: auto;
+#friendsApp.km-flat .km-navbar {
+  background-color: #fff;
 }
 
-#friendsApp .hide-text {
-    overflow: hidden;
-    text-indent: -999px;
+#friendsApp.km-flat .km-content {
+  color: #4d4d4d;
 }
 
-#friendsApp .center-text {
-    text-align: center;
+#friendsApp .hide-text {
+  text-indent: -999px;
+  overflow: hidden;
 }
 
 .disabled {
-    position: relative;
-    display: block;
+  position: relative;
+  display: block;
```

Comment:
If we target WP8.1+, we can probably just use pointer-events: none;
http://caniuse.com/#search=pointer-events